_monitoring-plugins/alerting/monitors.md

```diff
@@ -202,6 +220,15 @@ The line moves up and down as you increase and decrease the threshold. Once this
 
 Bucket-level monitors also require you to specify a threshold and value for your aggregation and timeframe, but you can use a maximum of five conditions to better refine your trigger. Optionally, you can also use a keyword filter to filter for a specific field in your index.
 
+Document-level monitors provide the added option to use tags that represent multiple queries connected by the logical OR operator.
+
+To create a multiple query combination trigger:
+
+1. Create a per document monitor with more than one query.
+2. Create the first query with field, operator and value. For example, set the query to search for the `region` field with either operator: "is" or "is not", and set the value "us-west-2".)
+3. Select **Add Tag** and give it a name.
+3. Create the second query and add the same tag to it.
+4. Now you can create the trigger condition and specify the tag name. This creates a combination trigger that checks two queries that both contain the same tag. The monitor checks both queries by a logical OR operation if either query's conditions are met, then it will generate the alert notification.
```

Comment:
Third sentence: Should "by a logical OR operation" be "with a logical OR operation"? The sentence also doesn't connect grammatically. Should it be "The monitor checks both queries with a logical OR operation, and if either..."?